src/components/Chat/InputMessageV2.tsx

```diff
@@ -65,7 +65,7 @@ const styles = StyleSheet.create({
   input: {
     flex: 1,
     backgroundColor: COLORS.gray100,
-    color: COLORS.white2,
+    color: COLORS.white,
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The change from `COLORS.white2` to `COLORS.white` for the text color might affect readability depending on the background color. If the background color is light, this could reduce contrast and make the text harder to read. Please ensure that this color change is in line with accessibility standards.
```diff
-    color: COLORS.white2,
+    color: COLORS.white,
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->